src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/InitialConsiderationController.java

```diff
@@ -40,6 +41,7 @@ public class InitialConsiderationController {
     private final InitialConsiderationService initialConsiderationService;
     private final DocumentManagementService documentManagementService;
     private final ReportDataService reportDataService;
+    private final CaseFlagsService caseFlagsService;
```

Comment:
give some space between private finals and private static finals. Maybe even move the latter above the former.